Expensify/k2-extension
# Implement Webpack for the build process

Fixes https://github.com/Expensify/Expensify/issues/192236

# Tests
1. Run `npm run web`
2. Load the unpacked extension into chrome
3. Verify the K2 dashboard works
